### DISCUSSION TITLE
Dark mode, advanced partitioning and buttons alignment

### DISCUSSION THREAD

**Discussion by leandrokemp** [rhinstaller/anaconda-webui#41] [General]
I have tested the new Anaconda Web UI installer. The interface looks pretty nice and modern.
However, I'd suggest some changes:

- There could be an option to select the theme to be used during the installer: Light or Dark mode. This selection could be remembered when the system is installed, so the DE would also be in the desired mode. This should be the first option to display once the installer launches;
- When selecting the disk, there could be a button called something like "Advanced partitioning", that would let us choose how to partition /, /home, /swap, etc. "Normal users" would not click on the advanced button and would proceed with the automatic partitioning;
- Footer buttons sequence could be reversed to "Quit, Back, Next" and right aligned as it's more intuitive for most people.

**Comment by jelly** [comment]
Cockpit supports Dark mode since 281. Cockpit appliation needs to update their `pkg/lib/` checkout to `281` and import `import "cockpit-dark-theme";`.

Normally Cockpit would switch the depending on the system theme. If you want to add a switcher in Anaconda the system style can be overriden with `localStorage.setItem('shell:style', 'dark')` (or light).
